app/editor/src/components/layout/DefaultLayout.tsx

```diff
@@ -23,21 +23,27 @@ interface ILayoutProps extends React.HTMLAttributes<HTMLDivElement> {
  */
 export const DefaultLayout: React.FC<ILayoutProps> = ({ name, children, ...rest }) => {
   const keycloak = useKeycloakWrapper();
+  const { setToken } = React.useContext(SummonContext);
   useToastError();
 
-  const state = React.useContext(SummonContext);
-  keycloak.instance.onTokenExpired = () => {
-    keycloak.instance
-      .updateToken(86400)
-      .then(function (refreshed: boolean) {
-        if (refreshed) {
-          state.setToken(keycloak.instance.token);
-        }
-      })
-      .catch(function () {
-        keycloak.instance.logout();
-      });
-  };
+  React.useEffect(() => {
+    keycloak.instance.onTokenExpired = () => {
+      keycloak.instance
+        .updateToken(240)
```

Comment:
The old timer was greater than the length of the refresh token itself.  Which didn't make sense to me.